### ISSUE TITLE
Accept folder as an argument

### ISSUE THREAD

**Issue by sobolevn** [ziima/polint#3]
Hi! Thanks for this package. It find its idea very promising.

I am trying to integrate it to our `django` workflow, https://github.com/wemake-services/wemake-django-template/issues/504

But, I am facing an issue: I can not pass a folder name to this tool. I have a base folder (`locale/`) that contains all the languages we use in our app. And I want to lint all of them at once.

That's how regular `django` translation directory looks like:

```
» tree locale
locale
└── ru
    └── LC_MESSAGES
        ├── django.mo
        └── django.po
└── it
    └── LC_MESSAGES
        ├── django.mo
        └── django.po
└── de
    └── LC_MESSAGES
        ├── django.mo
        └── django.po
```

And I want to lint all my `.po` files with a single command: `polint locale`.

Do you plan to support this feature?

**Comment by ziima** [comment]
In progress, thanks for the issue.

**Comment by ziima** [comment]
Released in 0.3

**Comment by sobolevn** [comment]
@ziima awesome! Thanks!
